docs/using_python_client_with_hazelcast.rst

```diff
@@ -1892,7 +1893,7 @@ method sends the predicate to all cluster members and merges the results
 coming from them.
 
 .. Note:: Predicates can also be applied to ``key_set`` and
-    ``entry_set`` of the Hazelcast IMDG’s distributed map.
+    ``entry_set`` of the map.
```

Comment:
Small note: Don't we need to use the article "a" here because we talk about a map in general? If it talks about a specific map(e.g map in the example), ignore this